src/pages/eco-score/index.tsx

```diff
@@ -148,33 +150,19 @@ const ecoScoreCards = [
   },
 ];
 
-export const countryNames = [
-  "en:belgium",
-  "en:denmark",
-  "en:france",
-  "en:germany",
-  "en:italy",
-  "en:netherlands",
-  "en:portugal",
-  "en:spain",
-  "en:sweden",
-  "en:switzerland",
-  "en:united-states",
-  "en:canada",
-  "en:australia",
-  "en:united-kingdom",
-];
+
 
 export default function EcoScore() {
   const { t } = useTranslation();
+  const localData= localSettings.fetch();
   const [searchParams, setSearchParams] = useSearchParams();
   const [selectedCountry, setSelectedCountry] = React.useState(
-    searchParams.get("cc") || countryNames[0],
+    searchParams.get("cc") || localData['country'] || countryNames[81].id,
   );
 
   React.useEffect(() => {
     setSearchParams({ cc: selectedCountry });
-  }, [selectedCountry]);
+  }, [selectedCountry, searchParams]);
```

Comment:
I don't see why adding this dependency

```suggestion
  }, [selectedCountry]);
```

Reply:
> I don't see why adding this dependency

Actually it's kind of important, if you go to https://hunger.openfoodfacts.org/eco-score?cc=en%3Afrance
and click `eco-score` in navbar, the query parameter disappears which is not an expected behavior. 
What you think?

Reply:
ok, why not :+1: